libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -92,13 +92,13 @@ export const fetchApplicationCommits = createAsyncThunk<Commit[], { applicationI
   }
 )
 
-export const fetchApplicationDeployments = createAsyncThunk<DeploymentHistoryApplication[], { applicationId: string }>(
-  'application/deployments',
-  async (data) => {
-    const response = await applicationDeploymentsApi.listApplicationDeploymentHistory(data.applicationId)
-    return response.data.results as DeploymentHistoryApplication[]
-  }
-)
+export const fetchApplicationDeployments = createAsyncThunk<
+  DeploymentHistoryApplication[],
+  { applicationId: string; silently: boolean }
+>('application/deployments', async (data) => {
+  const response = await applicationDeploymentsApi.listApplicationDeploymentHistory(data.applicationId)
```

Comment:
Here maybe you'll have to make if(silentely === undefined) silently = false something like this. Or simply `silently = !!silently`